policyengine_us/parameters/gov/states/ny/nyserda/drive_clean/drive_clean_high_msrp.yaml

```diff
@@ -0,0 +1,12 @@
+description: New York minimizes the Drive Clean point-of-sale rebate for high-price vehicles.
+metadata:
+  unit: miles
+  period: year
+  label: High Manufacturer's Suggested Price
+  reference:
+    - title: NYSERDA Drive Clean Implementation Manual
+      href: https://www.nyserda.ny.gov/All-Programs/Drive-Clean-Rebate-For-Electric-Cars-Program/Dealers/Dealer-Resources
```

Comment:
Could we link to the [implementation manual](https://www.nyserda.ny.gov/-/media/Project/Nyserda/Files/Programs/Drive-Clean-NY/implementation-manual.pdf) and add the relevant page to the hyper link (page where we can find the value of the parameter file)
e.g. `...manual.pdf#page=8` in this case